.evergreen/.evg.yml

```diff
@@ -2052,13 +2058,17 @@ buildvariants:
   tasks:
      - name: "gssapi-auth-test"
 
-- matrix_name: "perf"
-  matrix_spec:  { auth: "noauth", ssl: "nossl", jdk: "jdk17", version: ["latest"], topology: "standalone", os: "linux" }
-  batchtime: 1440 # run once a day
-  display_name: "Perf Tests ${version} "
-  tags: ["perf-variant"]
+- name: "perf"
+  display_name: Performance Tests
+  run_on: rhel90-dbx-perf-large
   tasks:
-     - name: "perf"
+    - name: "perf"
+      vars:
+        VERSION: "v6.0-perf"
```

Comment:
Driver evergreen tools exposes the patch pinned v6 server version 6.0.6 that can be referenced via the `v6.0-perf` version alias (the analogous perf-stable v7 version will be added later)